app/Services/CSVImportReader.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Services;
+
+use Illuminate\Support\LazyCollection;
+use App\Exceptions\ImportParserException;
+
+class CSVImportReader
+{
+    public function lines(string $path): LazyCollection
```

Comment:
I would have expected the `$path` as an argument to the constructor, rather than to the `lines()` function.

Reply:
We can do this, but this would mean we will not be able to leverage Laravel's 0 config DI. OR, well we could, but we would have to create a factory class or factory method, which seemed like redundant code to me at that point. What do you think?